apps/api/src/app/content-templates/usecases/compile-template/compile-template.spec.ts

```diff
@@ -61,6 +62,34 @@ describe('Compile Template', function () {
       })
     );
 
+    it('should allow the user to specify handlebars helpers'), async function () {
+      const result = await useCase.execute(
+        CompileTemplateCommand.create({
+          templateId: 'custom',
+          data: {
+            branding: {
+              color: '#e7e7e7e9',
+            },
+            message: 'hello world'
+          },
+          customTemplate: '<div>{{titlecase message}}</div>'
+        })
+      )
+    };
+
+    Handlebars.registerHelper('titlecase', function(string) {
+      string.trim().toLowerCase().replace(/\w\S*/g, (w) => (w.replace(/^\w/,
+      (c) => c.toUpperCase())));
```

Comment:
I can also see that this function is not returning nothing, so I'm not event sure that this will even work :) 
I also suggest adding a couple of expect statements to actually tests the behaviour of this helpers, since the bottom parts are not really testing it. 

Also if you could run prettier for this file :)